packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -391,6 +392,9 @@ class FinancialAssessment extends React.Component {
                                                         value={values.income_source}
                                                         error={touched.income_source && errors.income_source}
                                                         onChange={handleChange}
+                                                        onTouchEnd={e => {
+                                                            setFieldTouched(e.target.name, true);
+                                                        }}
```

Comment:
Does the fix only work with `onTouchEnd`? 
Would it work when done in `onChange`?

```suggestion
onChange={(e)=>{
setFIeldTouched(e.target.name, true);
handleChange(e);
}}
```

Reply:
No, I tried to use it onChange but it will only trigger validation if a user selects a new value in a dropdown

Reply:
`onTouchEnd` method gives almost identical behavior to what we have on desktop dropdown

Reply:
ok 👌🏻